collect_app/src/main/java/org/odk/collect/android/activities/InjectableActivity.java

```diff
@@ -0,0 +1,19 @@
+package org.odk.collect.android.activities;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+import dagger.android.AndroidInjection;
+
+/**
+ * @author James Knight
+ */
+
+public abstract class InjectableActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        AndroidInjection.inject(this);
```

Comment:
[Step 5](https://google.github.io/dagger/android.html) for injecting `Activity` objects says that `AndroidInjections.inject(this)` should be called prior to calling `super.onCreate(savedInstanceState)`

Reply:
Ah, interesting. Good catch.